gpflow/kernels/convolutional.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import tensorflow as tf
+
+from ..base import Parameter, positive
+from ..utilities.ops import square_distance
+from .base import Kernel
+from ..config import default_float
+
+
+class Convolutional(Kernel):
```

Comment:
I think it would make sense to go for a single `Convolutional` class which has `weights=np.ones()` or `weights=Parameter(np.ones())` depending on an `init` parameter. Quite easy to do and introduces less code duplication.

Reply:
@awav said this as well. Will do.